src/index.html

```diff
@@ -7,12 +7,74 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <section
+      data-qa="card"
+      class="card"
+    >
+      <div class="card__product">
+        <div class="card__product-img card__product-img--position">
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K (MNED2UA/A)"
+            class="card__img"
+          />
+        </div>
+
+        <div class="card__product-name">
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+        </div>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__product-stars">
+          <div class="card__stars">
+            <section class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
```

Comment:
The stars should be created using CSS, not HTML. You should have one div with a class for the number of active stars, and the stars themselves should be background images. This will make your HTML cleaner and your CSS more powerful.